package.json

```diff
@@ -8,6 +8,7 @@
     "check-types": "tsc",
     "preview": "vite preview",
     "lint": "eslint --max-warnings 0 'src/**/*.ts' 'src/**/*.tsx' && knip",
+    "lighthouse": "SCRIVITO_TENANT=d0a154d76edf2a7bd991fc658e700a1d SCRIVITO_ORIGIN=http://localhost:8080 PRERENDER_OBJ_ID=c2a0aab78be05a4e npm run prerender && precompress --include '(**).(css|html|js)' --types br dist-ssr && for run in $(seq 6); do lhci autorun && break; done",
```

Comment:
Regarding `for run in $(seq 6); do lhci autorun && break; done`: Do I understand this correctly, that lighthouse will try up to six times to reach the target score? Isn't `numberOfRuns` for that?

Reply:
Yes, but
* `numberOfRuns` will *always* collect N runs first and find the max score afterwards. This means **always** N times the CI duration.
* `numberOfRuns > 1` currently reports wrong numbers, see https://github.com/GoogleChrome/lighthouse-ci/issues/1067